src/externs.rs

```diff
@@ -220,3 +220,12 @@ pub unsafe extern "C" fn debug(str_ptr: *const u8, str_len: u32) {
 		r.borrow_mut().debug_log(msg);
 	});
 }
+
+#[derive(Default)]
+struct DummyExternal;
```

Comment:
why add `DummyExternal` here?

maybe a comment explaining why would be helpful

Reply:
The idea is to give an ability to provide some stub implementation of external calls. I don't know a better way how it could be done.

Reply:
oh, nwm, just a mistake. Thanks, should be removed.